src/lib/style/abstracts/_mixins.scss

```diff
@@ -99,19 +99,24 @@
     text-align: center;
     flex-grow: 1;
     padding: 5px 0px;
-    transition: border-bottom 300ms ease-in;
+    transition: border-bottom 200ms ease-in;
+    border-bottom: 3px solid transparent;
```

Comment:
All tabs have border-bottom to prevent re-sizing issues. Just make it transparent for inactive tabs